src/Mailer/Ownership/SendMailsForOwnershipEventHandler.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership;
+
+use Broadway\Domain\DomainMessage;
+use Broadway\EventHandling\EventListener;
+use CultuurNet\UDB3\Broadway\Domain\DomainMessageSpecificationInterface;
+use CultuurNet\UDB3\CommandHandling\ContextDecoratedCommandBus;
+use CultuurNet\UDB3\Http\AsyncDispatchTrait;
+use CultuurNet\UDB3\Mailer\Command\SendOwnershipRequestedMail;
+use CultuurNet\UDB3\Ownership\Events\OwnershipApproved;
+use CultuurNet\UDB3\Ownership\Events\OwnershipRejected;
+use CultuurNet\UDB3\Ownership\Events\OwnershipRequested;
+
+final class SendMailsForOwnershipEventHandler implements EventListener
+{
+    use AsyncDispatchTrait;
+
+    private ContextDecoratedCommandBus $mailerCommandBus;
+    private DomainMessageSpecificationInterface $isReplay;
+
+    public function __construct(
+        ContextDecoratedCommandBus $mailerCommandBus,
+        DomainMessageSpecificationInterface $domainMessageSpecification
+    ) {
+        $this->mailerCommandBus = $mailerCommandBus;
+        $this->isReplay = $domainMessageSpecification;
+    }
+
+    public function handle(DomainMessage $domainMessage): void
+    {
+        if ($this->isReplay->isSatisfiedBy($domainMessage)) {
+            // This is a replay, don't sent the email
```

Comment:
I think that is very clear with the `isReplay` property 😊

Reply:
You really hate comments. :-P

Reply:
Yes 😆 ... in this case the comment is just repeating the code.